src/sidebar/components/test/SearchInput-test.js

```diff
@@ -103,6 +117,171 @@ describe('SearchInput', () => {
     assert.isTrue(wrapper.exists('IconButton'));
   });
 
+  it('focuses search input when button pressed', () => {
+    fakeStore.isLoading.returns(false);
+    const wrapper = createSearchInput();
+    const inputEl = wrapper.find('input').getDOMNode();
+
+    wrapper.find('IconButton').props().onClick();
+
+    assert.equal(document.activeElement, inputEl);
+  });
+
+  describe('shortcut key handling', () => {
+    it('focuses search input when "/" is pressed outside of the component element', () => {
+      const wrapper = createSearchInput();
+      const searchInputEl = wrapper.find('input').getDOMNode();
+
+      document.body.dispatchEvent(
```

Comment:
This keyboard event dispatching could be extracted into a helper, eg. `pressShortcut(<key>, <extra properties for KeyboardEvent>)`.

Reply:
I poked at this a bit but didn't find that it immediately saved a whole lotta repetition...